src/Timer.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { TimerProps, TimerState } from './models/models';
+
+const START_TIME = 100;
+
+class Timer extends React.Component<TimerProps, TimerState> {
+
+  state = { time: { minutes: 1, seconds: 0 }, songCount: 1};
+
+  constructor(props: TimerProps) {
+    super(props);
+    this.startTimer = this.startTimer.bind(this);
+  }
+
+  componentDidMount() {
+    this.startTimer();
+  }
+
+  async startTimer() {
+    // Loop through a list of [0,1,2,3...START_TIME-1]
+    for (const num of Array(START_TIME).keys()) {
+      // Set a tick to occur every 1.000 * n seconds
+      await setTimeout(this.tick, 1000*num, num);
```

Comment:
Use system call setTimeout to schedule a call at ~1.000 * n seconds in the future. Do this for every n in our list of 0-n. So a tick should log about every second, though this setTimeout method technically only guarantees that the operation happens _before_ the specified number of ms, not _at_ that time. Havent tested in detail how accurate this timing is, but comparing 60s with my phone timer it seemed relatively consistent